platform-hub-web/src/app/kubernetes/kubernetes-clusters-form.component.js

```diff
@@ -15,12 +15,16 @@ function KubernetesClustersFormController($state, KubernetesClusters, logger) {
 
   ctrl.awsAccountIdRegex = '^[0-9]{12}$';
 
+  ctrl.separatorKeys = chipsHelpers.separatorKeys;
+  ctrl.separatorKeysHelpText = chipsHelpers.separatorKeysHelpText;
+
   ctrl.loading = true;
   ctrl.saving = false;
   ctrl.isNew = true;
   ctrl.cluster = null;
 
   ctrl.createOrUpdate = createOrUpdate;
+  ctrl.processAliasesChip = processAliasesChip;
```

Comment:
Is `processAliasesChip` defined somewhere? I can't find it

Reply:
It should be defined towards the bottom of this file.

Reply:
Oh yes my mistake! 👍